share/wrf_tsin.F

```diff
@@ -46,17 +49,35 @@ SUBROUTINE wrf_tsin ( grid , ierr )
    
              IF (istatus == 0) THEN
    
-                ! Ignore first three lines, which constitute a header
-                READ(UNIT=iunit, FMT='(1X)')
+                ! The header will be parsed to see if lat/lon or i/j is given...
                 READ(UNIT=iunit, FMT='(1X)')
+                READ(UNIT=iunit, FMT='(A32,A7,A2,A7,A1)') head1, JorLat, head2, IorLon, head3
```

Comment:
Since `head1`, `head2`, and `head3` don't seem to be used, I think we can discard these variables and change the `READ` line to look something like:
```fortran
                READ(UNIT=iunit, FMT='(32X,A7,2X,A7)') JorLat, IorLon
```

Reply:
@mgduda 
Agreed. I'll change it.